externe_connecties/OSF_connectie.py

```diff
@@ -0,0 +1,29 @@
+from config.configurations import OSF_API_GEEF_FOUTEN_URL
+
+import requests
+
+
+def OSF_aanroep(jsonbestandsnaam):
+    """
+    Regelt de aanroep naar de externe API van Open State Foundation (OSF).
+    Zend JSON bestand naar OSF API waar controle plaatsvindt en die JSON bestand met fouten element terug stuurt.
+    """
+    bestanden = {'file': open(jsonbestandsnaam)}
+
+    # try:
+    response = requests.post(OSF_API_GEEF_FOUTEN_URL, files=bestanden)
+
+    # except FileNotFoundError:
+    #     print("Het geuploade json bestand bestaat niet of kan niet gevonden worden.")
+    #     codefouten.append("Het geuploade json bestand bestaat niet of kan niet gevonden worden.")
+    #     _logger.info("Het geuploade json bestand bestaat niet of kan niet gevonden worden.")
+    return response
+
+def geef_fouten(jsonbestandsnaam):
```

Comment:
Is tijdelijk, want uiteindelijk willen we niet alleen de fouten, maar alle informatie uit de call, lijkt me

Reply:
klopt uiteindelijk willen we meer informatie, maar voor de fouten kan dit volgens mij blijven. Anyway, het is makkelijk aangepast in de toekomst toch?